### ISSUE TITLE
React + Relay: yarn start fails after restructuring project in /1-getting-started/

### ISSUE THREAD

**Issue by MikaelCarpenter** [howtographql/howtographql#144]
## Issue
In `/react-relay/1-getting-started/` we create a new project and restructure the project to add `src/components` and `src/styles` folders. We then move some files into them. When we later try to run `yarn start` we get an error saying that `index.js` cannot find the files we moved

## Reproduce
- `create-react-app hackernews-react-relay`
- `cd hackernews-react-relay`
- Update file structure like so:
```
├── README.md
├── node_modules
├── project.graphcool
├── package.json
├── public
│   ├── favicon.ico
│   ├── index.html
│   └── manifest.json
├── src
│   ├── App.test.js
│   ├── components
│   │   └── App.js
│   ├── index.js
│   ├── logo.svg
│   ├── registerServiceWorker.js
│   └── styles
│       ├── App.css
│       └── index.css
└── yarn.lock
```
- `yarn start`
- You should see error saying `index.js` cannot find `.App.js` and `.index.css`

## Proposed Solution
Add a small snippet after the file restructuring that notifies the student that they need to adjust their import statements accordingly. 

**PR incoming with that solution**

**Comment by marktani** [comment]
Thanks, that's amazing!

**Comment by marktani** [comment]
Thanks @MikaelCarpenter, fixed in #145.
